app/models/mixins/retirement_mixin.rb

```diff
@@ -231,15 +231,36 @@ def valid_zone?
     respond_to?(:my_zone) && my_zone.present?
   end
 
+  def system_context_requester
+    if try(:evm_owner_id).present?
```

Comment:
Do you need the `present?` check?  It should be `nil` or an ID, right?

Reply:
heh. don't need the try either